examples/webrtc-client-example/README.md

```diff
@@ -1,31 +1,37 @@
-# Squawk WebRTC Client 
+![alt text](https://raw.githubusercontent.com/Benzinga/benzinga-python-client/master/logo/Benzinga_Logo-navy.png)
 
-## Overview
+# benzinga-squawk-client
 
-This Squawk WebRTC Client demonstrates how to connect to squawk over WebRTC using signaling messages documented at https://docs.benzinga.io/benzinga/squawk-v3.html#using-webrtc
+## Overview
+Squawk is a realtime broadcast service from Benzinga which includes important headlines, price movement, and rumors as stories develop to give traders and investors news in the fastest and most convenient form. This repo icludes sample applications, which aims to elaborate how you can connect to Squawk.
 
-- [npm](https://nodejs.org/en/)
-- [yarn](https://classic.yarnpkg.com/en/docs/install#debian-stable)
-- Squawk WebSocket address. Contact the Benzinga licensing team at  licensing@benzinga.com for more details.
-- JWT token. For authentication, squawk uses JWT. We recommend using asymmetric JWT. For that, you need to share your the corresponding public key with Benzinga. Please get in touch at licensing@benzinga.com for sharing the public key or more information on authentication. 
 
-## How to Build and Run
+## Connecting to Squawk
 
-Once you have shared the public key for JWT with Benzinga and got the squawk address to test for, replace corresponding values in .env file as `SQUAWK_ADDR` and `JWT_TOKEN`. 
+One can connect to squawk in the following ways:
 
-Use the yarn package manager to run, test, and build the app.
+1. **Connect directly to squawk through WebRTC:** With this option, you can connect directly to squawk from your web application and hear it from WebRTC supported browsers. More Information can be found at [https://docs.benzinga.io/benzinga/squawk-v4.html#using-webrtc](https://docs.benzinga.io/benzinga/squawk-v4.html#using-webrtc)
 
```

Comment:
Okay, as mentioned above, this update (link to squawk-v4 docs) should be part of the parent directory's README.